packages/core-contracts/test/utils/Ownable.test.js

```diff
@@ -0,0 +1,196 @@
+const { ethers } = hre;
+const assert = require('assert/strict');
+
+const { ZERO_ADDRESS } = require('../test-libs/test-lib');
+
+describe('Ownable', () => {
+  describe('Constructor / Deployment', () => {
+    it('Should revert when no argument passed to Ctor', async () => {
```

Comment:
No need to check for incorrect ABI usage. Just for invalid constructor parameters.